companies/dm-drogerie.json

```diff
@@ -0,0 +1,29 @@
+{
+    "slug": "dm-drogerie",
+    "relevant-countries": [
+        "de",
+        "at",
+        "cz",
+        "it",
+        "ro",
+        "hu",
+        "rs",
+        "ba",
+        "hr",
+        "si",
+        "sk",
+        "mk",
+        "bg"
+    ],
+    "name": "dm-drogerie markt GmbH + Co. KG",
+    "address": "Am dm-Platz 1\n76227 Karlsruhe\nDeutschland",
+    "phone": "+49 721 55920",
+    "fax": "+49 0721 552213",
+    "email": "servicecenter@dm.de",
```

Comment:
They hid the proper email right at the bottom of the privacy policy page `datenschutz@dm.de` :D Let's use that one instead.